### PR TITLE
feat(assets): Add content icons for xbd and xdw

### PR DESCRIPTION
**Pull request by ConradJChan**

### CODE REVIEW

File: src/illustration/Files.tsx [hunk 1]
```diff
@@ -1,64 +0,0 @@
...
- * manually in contributor pull requests.
- *
- * If you have problems with this component:
- * - https://github.com/box/box-ui-elements/issues/new?template=Bug_report.md
- *
- * If there are missing features in this component:
- * - https://github.com/box/box-ui-elements/issues/new?template=Feature_request.md
- */
-
-const Files = (props: SVGProps) => (
```
Comment (jstoffan) [inline src/illustration/Files.tsx:16]: Is this no longer used anywhere?

File: src/icon/line/Report16.stories.tsx [hunk 2]
```diff
@@ -1,13 +0,0 @@
...
-
-import Report16 from './Report16';
-
-export const report16 = () => <Report16 />;
-
-export default {
-    title: 'Icon|Line|Report16',
-    component: Report16,
-    parameters: {
-        notes: "`import Report16 from 'box-ui-elements/es/icon/line/Report16';`",
```
Comment (jstoffan) [inline src/icon/line/Report16.stories.tsx:11]: Does this no longer exist? I don't see it being removed in this PR.

File: src/icon/fill/SignFieldCheckBox16.js.flow [hunk 3]
```diff
@@ -19,9 +19,9 @@ import type { Icon } from '../../icons/flowTypes';
 const SignFieldCheckBox16 = (props: Icon) => (
     <AccessibleSVG width={16} height={16} viewBox="0 0 16 16" {...props}>
         <path
+            d="M11.802 2c.765 0 1.042.08 1.321.229.28.15.499.369.648.648.15.28.229.556.229 1.32v7.605c0 .765-.08 1.042-.229 1.321-.15.28-.369.499-.648.648-.28.15-.556.229-1.32.229H4.197c-.765 0-1.042-.08-1.321-.229a1.558 1.558 0 01-.648-.648c-.15-.28-.229-.556-.229-1.32V4.197c0-.765.08-1.042.229-1.321.15-.28.369-.499.648-.648.28-.15.556-.229 1.32-.229h7.605zm-.752 2.143a.761.761 0 00-.396.097.961.961 0 00-.309.335l-3.178 5.24-1.598-2.004a.763.763 0 00-.623-.307.769.769 0 00-.575.235.813.813 0 00-.228.59c0 .105.017.205.051.3.035.094.096.194.185.3l2.088 2.586a.888.888 0 00.713.342c.326 0 .574-.129.743-.387l3.762-5.962c.06-.096.104-.19.131-.281a.925.925 0 00.041-.264.803.803 0 00-.23-.586.776.776 0 00-.577-.234z"
```
Comment (jstoffan) [inline src/icon/fill/SignFieldCheckBox16.js.flow:22]: Any concerns from the Sign team on these changes?

File: src/icon/line/Report16.stories.tsx (hunk 2 above)
Comment (ConradJChan) [reply src/icon/line/Report16.stories.tsx:11]: Looks like `Report16` got renamed to `AlertBubble16` which has it's corresponding stories file

**Review by jstoffan — APPROVED** [review]

**Pull request merged**